todo.asm

```diff
@@ -250,6 +250,9 @@ add_todo:
    push rdi ;; void *buf [rsp+8]
    push rsi ;; size_t count [rsp]
 
+   funcall3 find_char, [rsp+8], [rsp], 61 ;equal sign
+   inc rax
```

Comment:
Do I understand correctly that you are trying to parse form-data in `add_todo`?

Reply:
`todo=test`
        ^find_char result
then increase rax by one to skip the equal sign.
since you only submit a one todo at a time i believe this should be fine

Reply:
I want to keep `add_todo` unaware of the specific of HTTP protocol, so I can use it in case I add more ways to interact with the application. 

Please parse it outside of `add_todo`, just like it was implemented originally, before I decided to get rid of the HTML forms, cause 
a simple string in the HTTP body is easier to parse https://github.com/tsoding/todo.asm/commit/101ceb774339c2d2a07724e80710ecd595c7954d#diff-834d933011598994721ae31992e3434fd9a5dd8582af466bd497e5b89ac332f5L129-L136